reactDashboard/src/components/DataDefinitions/DataDefinitions.css

```diff
@@ -8,20 +7,29 @@
   height: 30px;
 }
 
+.darkmode .data-definitions .data-accordion-toggle-down {
```

Comment:
These are the same colours as lightmode - can this block be deleted?

Reply:
✅